packages/make-styles/src/renderer/createDOMRenderer.test.ts

```diff
@@ -0,0 +1,57 @@
+import { CSSRulesByBucket } from '../types';
+import { createDOMRenderer } from './createDOMRenderer';
+import { makeStylesRendererSerializer } from '../common/snapshotSerializers';
+
+function createFakeDocument(): Document {
+  const doc = document.implementation.createDocument('http://www.w3.org/1999/xhtml', 'html', null);
+  doc.documentElement.appendChild(document.createElementNS('http://www.w3.org/1999/xhtml', 'head'));
+
+  return doc;
+}
```

Comment:
```suggestion
```

This will explode during Jest upgrade (#20716), so let's remove it, https://github.com/microsoft/fluentui/pull/20716#discussion_r759260665.

Reply:
removed and added a cleanup step in `beforeEach` since makeStyles will keep dumping style tags